spec/comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  comment = Comment.new(author_id: 1, post_id: 1)
+
+  it 'Reutrns author id' do
+    expect(comment.author_id).to eql 1
+  end
+
+  it 'Reutrns post id' do
+    expect(comment.post_id).to eql 1
+  end
+
+  it 'checks comments_counter method' do
+    User.new(name: 'Ahmad', photo: 'www.iamge.com', bio: 'Full Stack Developer', posts_counter: 5)
+    post = Post.new(author_id: 1, title: 'New', text: 'First post', comments_counter: 3, likes_counter: 2)
+    comment = Comment.new(author_id: 1, post:)
```

Comment:
- [ ] [REQUIRED I currently get a syntax error from this file while trying to test. Kindly ensure that you provide a value for the post-argument. By providing a valid value for the post-argument, the syntax error should be resolved. 

<img width="623" alt="Screenshot 2023-05-19 at 17 12 14" src="https://github.com/batoorsohail/blog-app/assets/58692590/961e459b-8f3d-421b-a850-647461a62cf1">